addons/account/views/report_invoice.xml

```diff
@@ -9,7 +9,7 @@
                         <div name="invoice_address" class="col-xs-5 col-xs-offset-7">
                             <address t-field="o.partner_id"
                                 t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
-                            <div t-if="o.partner_id.vat" class="mt16">VAT: <span t-field="o.partner_id.vat"/></div>
+                            <div t-if="o.partner_id.vat" class="mt16"><span t-field="o.company_id.country_id.vat_label"/>: <span t-field="o.partner_id.vat"/></div>
```

Comment:
either you give a default value in the field definition, either you need to display o.company_id.country_id.vat_label or 'TIN'